workflow/rocoto/gfs_tasks.py

```diff
@@ -856,7 +856,9 @@ def _fcst_forecast_only(self):
             dep_dict = {'type': 'task', 'name': f'{self.cdump}{wave_job}'}
             dependencies.append(rocoto.add_dependency(dep_dict))
 
-        if self.app_config.do_aero and self.cdump in self.app_config.aero_fcst_cdumps:
+        if self.app_config.do_aero and \
+           self.cdump in self.app_config.aero_fcst_cdumps and \
+           self._base['EXP_WARM_START'] in ['.false.']:
```

Comment:
should this not be `False` instead of `'.false.'` ? Even when EXP_WARM_START is set to `'.false.'` in config.base it gets set to `False`.  Printing out `self._base['EXP_WARM_START']` shows `False`

Reply:
excellent catch!